src/sbt-test/sbt-cake/simple/src/main/scala/Hello.scala

```diff
@@ -1 +1 @@
-object TerribleFormatting { def foo = "bar" }
+object TerribleFormatting{def foo=   "bar"}
```

Comment:
Isn't it better to add some comment on top of this code explaining that this is malformatted intentionally?